src/libsync/syncfilestatustracker.cpp

```diff
@@ -68,6 +68,12 @@ static inline bool showWarningInSocketApi(const SyncFileItem& item)
         || status == SyncFileItem::Restoration;
 }
 
+static inline bool showSyncInSocketApi( const SyncFileItem& item)
+{
+    const auto inst = item._instruction;
```

Comment:
Is it really worth making a function for a one liner?  And why introduce this variable inst?

And anyway, what's so special about CSYNC_INSTRUCTION_NEW ?  Then it should also be the case for CSYNC_INSTRUCTION_SYNC should it not? 

Unless I misunderstand this.